YMChat/src/main/java/com/yellowmessenger/ymchat/YMChat.java

```diff
@@ -64,11 +68,17 @@ public void closeBot() {
 
     public void emitEvent(YMBotEventResponse event) {
         if (event != null) {
-            if (listener != null)
-                listener.onSuccess(event);
+            if(botCloseEventListener!= null && event.getCode().equals("bot-closed")){
```

Comment:
can event.getCode() be null? If yes, lets put a null check for this as well.

Reply:
added null check